### PR TITLE
Anime Facts Application

### PR DESCRIPTION
**Pull request by 10-menachi**
Created:
- Searchbar for users to search their favorite anime
- Homepage for users to view their favorite anime
- Dynamic routing so that a user can view details about a specific anime
- App deployment on Render
- Tests for components and redux slice
- Professional README
- License

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+[Live Demo](https://silver-crisp-2f216f.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE VIDEO -->
+
+## 🚀 Live Video <a name="live-video"></a>
+
+[Live Video](https://www.loom.com/share/08e77e4206e542e8b92b4405021b9caa?sid=620f7dbb-3347-4b71-973a-923c4f523d27)
```
Comment (nahnahdev) [inline README.md]: - [x] Great job @10-menachi There are just a few changes I will suggest you make and you'll be on your way to the next milestone :wink: 
- [ ] Great job adding the Presentation video :clap: However, I noticed that the video doesn't contain a highlight of a piece of the code that you find interesting or are proud of making. :+1: 
- [ ] Also, the design you were given was a mobile design but in your video, you didn't show the mobile view you only showed the desktop view. So kindly redo the video and ensure that it is in mobile view and also contains the earlier comment :wink:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me this opportunity to learn and build this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (nahnahdev) [inline README.md]: ##### **Creative Commons attribution**

- [x] Please ensure that the "Acknowledgements" section of your README gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). :100:

File: src/App.jsx [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
 const App = () => (
   <div>
-    App
+    <SearchBar />
+    <Routes>
+      <Route path="/" element={<AnimeListView />} />
+      <Route path="/anime/:id" element={<AnimeView />} />
+    </Routes>
   </div>
 );
```
Comment (nahnahdev) [inline src/App.jsx]: - [x] Your App looks great. However, per the project requirements, you need to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)) including:
   * Typography: font face, size, and weight
   * Layout: composition and space between elements.
- [ ] However, in the mobile view of your app, the homepage is not following the layout as seen in the Behance design. 
- You can see that the Behance design has the contents of the page split in two and the boxes containing each country don't have spaces between. Aside from this, each box has an arrow that when clicked on takes a user to the details page but this wasn't incorporated in your design so kindly include them as well :wink: 
- I also noticed that you are missing the `navbar` as seen in the Behance design so kindly include that as well :wink: 

| Your Design | The Behance Design |
|--------|--------|
| image |image |

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @10-menachi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :fire: 
- LInters well setup and passing :heavy_check_mark: 
- App deployed as expected :heavy_check_mark: 
- Project works without errors :heavy_check_mark: 
- All required tests included in the project and passing :heavy_check_mark: 
- Most project requirements met :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @10-menachi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Great job in fixing all the comments raised in the previous code review :rocket: 


### Optional
- [ ] It would be nice going forward, we follow the correct Git-flow workflow, **NOTE**: 1. For group capstone projects, we are required to use the GitHub workflow where you create the PR branch from a descriptive dev branch into the master branch 2. For solo capstone projects, we are required to use the Git-flow workflow


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @10-menachi,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**